From 8b92f46ab371ec51dac2b73c768173cd63126402 Mon Sep 17 00:00:00 2001
From: Alan Coopersmith <alan.coopersmith@oracle.com>
Date: Tue, 27 Oct 2015 16:19:51 -0700
Subject: [PATCH:macros 1/2] XORG_MANPAGE_SECTIONS: limit SysV man page
 sections to Solaris 2.0-11

All others (including other SunOS/Solaris releases) use the
traditional Bell Labs / BSD / Linux section numbering.

(The [[01]] is an m4-quoted shell character set of "[01]" to make sure
 autoconf doesn't lose the [] when generating the configure script.)

Signed-off-by: Alan Coopersmith <alan.coopersmith@oracle.com>
Reviewed-by: Jeremy Huddleston Sequoia <jeremyhu@apple.com>
---
 xorg-macros.m4.in | 22 +++++++++++++---------
 1 file changed, 13 insertions(+), 9 deletions(-)

diff --git a/xorg-macros.m4.in b/xorg-macros.m4.in
index f160a40..9b3c503 100644
--- a/xorg-macros.m4.in
+++ b/xorg-macros.m4.in
@@ -1,6 +1,6 @@
 dnl @configure_input@
 dnl
-dnl Copyright (c) 2005, 2006, Oracle and/or its affiliates. All rights reserved.
+dnl Copyright (c) 2005, 2015, Oracle and/or its affiliates. All rights reserved.
 dnl
 dnl Permission is hereby granted, free of charge, to any person obtaining a
 dnl copy of this software and associated documentation files (the "Software"),
@@ -130,8 +130,9 @@ fi
 
 if test x$FILE_MAN_SUFFIX = x    ; then
     case $host_os in
-	solaris*)	FILE_MAN_SUFFIX=4  ;;
-	*)		FILE_MAN_SUFFIX=5  ;;
+	# Solaris 2.0 - 11 use SysV man page sections
+	solaris2.?|solaris2.1[[01]])	FILE_MAN_SUFFIX=4  ;;
+	*)				FILE_MAN_SUFFIX=5  ;;
     esac
 fi
 if test x$FILE_MAN_DIR = x    ; then
@@ -140,8 +141,9 @@ fi
 
 if test x$MISC_MAN_SUFFIX = x    ; then
     case $host_os in
-	solaris*)	MISC_MAN_SUFFIX=5  ;;
-	*)		MISC_MAN_SUFFIX=7  ;;
+	# Solaris 2.0 - 11 use SysV man page sections
+	solaris2.?|solaris2.1[[01]])	MISC_MAN_SUFFIX=5  ;;
+	*)				MISC_MAN_SUFFIX=7  ;;
     esac
 fi
 if test x$MISC_MAN_DIR = x    ; then
@@ -150,8 +152,9 @@ fi
 
 if test x$DRIVER_MAN_SUFFIX = x    ; then
     case $host_os in
-	solaris*)	DRIVER_MAN_SUFFIX=7  ;;
-	*)		DRIVER_MAN_SUFFIX=4  ;;
+	# Solaris 2.0 - 11 use SysV man page sections
+	solaris2.?|solaris2.1[[01]])	DRIVER_MAN_SUFFIX=7  ;;
+	*)				DRIVER_MAN_SUFFIX=4  ;;
     esac
 fi
 if test x$DRIVER_MAN_DIR = x    ; then
@@ -160,8 +163,9 @@ fi
 
 if test x$ADMIN_MAN_SUFFIX = x    ; then
     case $host_os in
-	solaris*)	ADMIN_MAN_SUFFIX=1m ;;
-	*)		ADMIN_MAN_SUFFIX=8  ;;
+	# Solaris 2.0 - 11 use SysV man page sections
+	solaris2.?|solaris2.1[[01]])	ADMIN_MAN_SUFFIX=1m ;;
+	*)				ADMIN_MAN_SUFFIX=8  ;;
     esac
 fi
 if test x$ADMIN_MAN_DIR = x    ; then
-- 
2.7.4

